ui/server/src/main/scala/pl/touk/nussknacker/ui/api/DefinitionResources.scala

```diff
@@ -41,20 +41,18 @@ class DefinitionResources(modelDataProvider: ProcessingTypeDataProvider[ModelDat
       processingTypeDataProvider.forType(processingType).map { processingTypeData =>
         //TODO maybe always return data for all subprocesses versions instead of fetching just one-by-one?
         pathEndOrSingleSlash {
-          post { // POST - because there is sending complex subprocessVersions parameter
-            entity(as[Map[String, Long]]) { subprocessVersions =>
-              parameter('isSubprocess.as[Boolean]) { (isSubprocess) =>
-                val subprocesses = subprocessRepository.loadSubprocesses(subprocessVersions)
-                complete(
-                  UIProcessObjectsFactory.prepareUIProcessObjects(
-                    processingTypeData.modelData,
-                    processingTypeData.deploymentManager,
-                    user,
-                    subprocesses,
-                    isSubprocess,
-                    processCategoryService)
-                )
-              }
+          get {
```

Comment:
where is change post->get in UI client ? or is it transparent in that part ?

Reply:
It's in `HttpService.ts`

Reply:
Somehow I missed this file